mantidproject/mantid
# Project Recovery: Graceful handling of group workspaces

SHOULD ONLY BE MERGED AFTER #23451 and #23447

**Description of work.**
Removed the previous way of handling GroupWorkspaces as it will no longer after required pull requests be required as it is handled properly now.

**To test:**
- Group two workspaces
- Wait for project recovery to save
- Run Segfault
- Load up mantid
- Attempt recovery
- The group should be there with it's respective workspaces

Fixes #23452 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
